app/controllers/defendants_controller.rb

```diff
@@ -13,6 +13,16 @@ def show
                    defendant_path(defendant.arrest_summons_number || defendant.national_insurance_number)
   end
 
+  def remove_link
+    defendant.update(
+      user_name: 'example',
```

Comment:
If is this is intended to be the user that unlinked, can we not just send `current_user.email` (what is the expected format of user_name (i would push for email address if possible but think there may be issues with that related to MAATs username logic)

Reply:
From looking at the adapter, I think [user names are limited to ten characters](https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/app/contracts/unlink_defendant_contract.rb#L17-L19) - I did have this as email originally.

I wasn't sure what usernames are most useful, so wanted to skip that discussion, get this merged, and worry about it in a follow-up! Is your preference to create tickets for things like that?

Reply:
sounds good 👍